presentation/src/main/java/com/safetyheads/akademiaandroida/presentation/ui/fragments/writetous/WriteToUsFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.safetyheads.akademiaandroida.writetous
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.safetyheads.akademiaandroida.presentation.databinding.FragmentWriteToUsBinding
+import com.safetyheads.akademiaandroida.presentation.ui.utils.EmailValidator
+import com.safetyheads.akademiaandroida.presentation.ui.utils.FullNameValidator
+import com.safetyheads.akademiaandroida.presentation.ui.utils.PhoneNumberValidator
+
+
+class WriteToUsFragment : Fragment() {
```

Comment:
This fragment should base on `BottomSheetDialogFragment` like ProfileBottomSheetDialogFragment